src/modules/menu/components/activeLayers/MenuActiveLayersListItem.vue

```diff
@@ -213,67 +226,75 @@ function duplicateLayer() {
+                    class="menu-layer-transparency-slider ms-2 me-4"
+                    type="range"
+                    min="0.0"
+                    max="1.0"
+                    step="0.01"
+                    :value="1.0 - layer.opacity"
+                    :data-cy="`slider-transparency-layer-${id}-${index}`"
+                    @mouseup="onTransparencyCommit"
+                    @input="debounceTransparencyChange"
+                />
+                <button
+                    v-if="hasMultipleTimestamps"
+                    class="btn d-flex align-items-center"
+                    :class="{ 'btn-lg': !compact }"
+                    :data-cy="`button-duplicate-layer-${id}-${index}`"
+                    data-tippy-content="duplicate_layer"
+                    @click.prevent="duplicateLayer()"
+                >
+                    <FontAwesomeIcon :icon="['far', 'copy']" />
+                </button>
+                <button
+                    ref="layerUpButton"
+                    class="btn-layer-up-down btn d-flex align-items-center"
+                    :class="{ 'btn-lg': !compact }"
+                    :disabled="isTopLayer"
+                    :data-cy="`button-raise-order-layer-${id}-${index}`"
+                    @click.prevent="emit('moveLayer', index, index + 1)"
+                >
+                    <FontAwesomeIcon icon="arrow-up" />
+                </button>
+                <button
+                    ref="layerDownButton"
+                    class="btn-layer-up-down btn d-flex align-items-center"
+                    :class="{ 'btn-lg': !compact }"
+                    :disabled="isBottomLayer"
+                    :data-cy="`button-lower-order-layer-${id}-${index}`"
+                    @click.prevent="emit('moveLayer', index, index - 1)"
+                >
+                    <FontAwesomeIcon icon="arrow-down" />
+                </button>
+                <button
+                    v-if="showLayerDescriptionIcon"
+                    class="btn d-flex align-items-center"
+                    :class="{ 'btn-lg': !compact }"
+                    :data-cy="`button-show-description-layer-${id}-${index}`"
+                    @click="showLayerDescriptionPopup"
+                >
+                    <FontAwesomeIcon icon="info-circle" />
+                </button>
+            </div>
+            <div v-if="isLayerKml" v-show="showLayerDetail" class="menu-layer-item-details mt-n2">
+                <label class="menu-layer-transparency-title text-nowrap me-2">
+                    {{ t('vector_feedback_select_style') }}
+                </label>
+                <select v-model="currentKmlStyle" class="form-select form-select-sm">
+                    <option v-for="style in allKmlStyles" :key="style" :value="style">
+                        {{ t(`kml_style_${style.toLowerCase()}`) }}
+                    </option>
+                </select>
```

Comment:
We should be careful with the drop down box, we currently have 3 different types:

- select type like here and a few other places
- Home made pure bootstrap like in import tool and search bar
- tippy like in drawing menu style size selectors

Perso I don't like the look and feel of the select type as the dropdown box is squared and don't match the rest of the styling. I also find the tippy not so nice, because the dropdown placement is not always at the bottom.

Can we not make the <select> drop down look like the rest of the box, rounded corner with a grey border instead of black ? Or should we implement our own select that match the style of the search bar and import map tools ?

Reply:
Otherwise some top margin to the new line in the settings would be nicer.

Reply:
I've switch to the DropdownButton component, might be better for consistency yes...

Reply:
Well now in the menu we have 3 types, the DropdownButton component is only used in drawing currently.
Might suggestion for this PR would be to also change the downdown in the print menu so we have in the menu only two styles the tippy dropdown and the import tool home made advanced dropdown with button group.
Then in a separate PR we could simply change one component `DropdownButton` to match the same style as the sear bar and import map tools and we have one single style over the whole app 😄

Reply:
https://jira.swisstopo.ch/browse/PB-1048
I've created a task to do that later on, I don't want to spend more time on this right now (quite enough on my table)